modules/rosa-hcp-sharing-vpc-cluster-creation.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-hcp-shared-vpc-config.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sharing-vpc-cluster-creation_{context}"]
+= Step Four - Cluster Creator: Creating your cluster in a shared VPC
+To create a cluster in a shared VPC, complete the following steps.
+
+[NOTE]
+====
+Installing a cluster in a shared VPC is supported only for OpenShift 4.12.34 and later, 4.13.10 and later, and all future 4.y-streams.
+====
+
+image::372_OpenShift_on_AWS_persona_worflows_0923_4.png[]
+.Prerequisites
+
+* You have the hosted zone IDs from the *VPC Owner*.
+* You have the AWS region from the *VPC Owner*.
+* You have the subnet IDs from the *VPC Owner*.
+* You have the `Route 53 role` ARN from the *VPC Owner*.
+* You have the `VPC endpoint role` ARN from the *VPC Owner*.
+
+.Procedure
+* In a terminal, enter the following command to create the shared VPC:
++
+[source,terminal]
+----
+rosa create cluster --cluster-name <cluster_name> --sts --operator-roles-prefix <prefix> --oidc-config-id <oidc_config_id> --region us-east-1 --subnet-ids <subnet_ids> --hcp-internal-communication-hosted-zone-id <private_hosted_zone_ID> --ingress-private-hosted-zone-id <local_hosted_zone_ID> --route53-role-arn <route_53_role_arn> vpc-endpoint-role-arn <vpc_endpoint_role_arn> --base-domain <dns-domain> --additional-allowed-principals --hosted-cp
+----
++
+include::snippets/rosa-long-cluster-name.adoc[]
```

Comment:
We should probably take this snippet out here or move it because they need to know this to create the hosted zones.